### PR TITLE
Without overlapping

### PR DESCRIPTION
**Pull request by VeerUP**
Add withoutOverlapping() method (https://laravel.com/docs/5.5/scheduling#preventing-task-overlaps)

### CODE REVIEW

File: Schedule.php [hunk 1]
```diff
@@ -17,11 +20,29 @@ class Schedule extends Component
...
      */
     public $cliScriptName = 'yii';
 
+    /**
+     * Schedule constructor.
+     * @param array $config
+     */
+    public function __construct(array $config = [])
+    {
+        $this->_mutex = Yii::$app->has('mutex') ? Yii::$app->get('mutex') : (new FileMutex());
```
Comment (zhukovra) [inline Schedule.php]: @VeerUP I think that is needed to add some notes to README about mutex configuration with link to [component documentation](http://www.yiiframework.com/doc-2.0/yii-mutex-mutex.html)

**Comment by VeerUP** [comment]
Fixed

**Comment by zhukovra** [comment]
@gietos I think that it could be merged after squashing. Do you?

**Comment by omnilight** [comment]
Merged, thank you

**Pull request merged**